spec.bs

```diff
@@ -185,44 +185,59 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>search componen
 Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component</dfn>, a [=component=], which must be set upon creation.
 
 <dl class="domintro non-normative">
-  <dt><code>{{URLPattern}} . {{URLPattern/protocol}}</code>
+  <dt><code>|urlPattern| = new {{URLPattern/constructor(input, baseURL)|URLPattern}}(|input|, |baseURL|)</code></dt>
+  <dd class=allow-2119>
+    Constructs a new URLPattern object.  The |input| may be a {{URLPatternInit}} dictionary or a string.  If |input| is a string, then an optional |baseURL| string may also be provided.
```

Comment:
```suggestion
    Constructs a new {{URLPattern}} object.  The |input| may be a {{URLPatternInit}} dictionary or a string.  If |input| is a string, then an optional |baseURL| string may also be provided.
```

Reply:
I prefer not to mention the concept "dictionary" (it's not something web developers are familiar with) or reference IDL constructs which don't have their own domintros. So I'd suggest something like "input can be an object containing protocol, username, password, hostname, port, pathname, search, hash, and baseURL values which are used to construct the pattern" for the constructor, and "which are matched against the pattern" for the others.